azkeys/keyvault.go

```diff
@@ -43,8 +43,6 @@ func (kv *KeyVault) GetKeyByKID(
 	log := logger.Sugar.FromContext(ctx)
 	defer log.Close()
```

Comment:
The request-scoped logger can go as well?

Reply:
yes it can. thanks!